database/seeders/DatabaseSeeder.php

```diff
@@ -15,11 +15,13 @@ class DatabaseSeeder extends Seeder
     public function run()
     {
         $this->call([
-            TrackSeeder::class,
             AdminSeeder::class,
             UserSeeder::class,
             ChallengeSeeder::class,
             SubmissionSeeder::class,
+            TeamSeeder::class,
+            TrackSeeder::class,
```

Comment:
TrackSeeder should be run **before** the **UserSeeder** and **ChallengeSeeder** as they depend on it